java/system-test/system-test-suite/src/test/java/sleeper/systemtest/suite/PartitionSplittingIT.java

```diff
@@ -48,6 +51,9 @@ public class PartitionSplittingIT {
     @RegisterExtension
     public final ReportingExtension reporting = ReportingExtension.reportIfTestFailed(
             sleeper.reportsForExtension().partitionStatus());
+    @RegisterExtension
+    public final PurgeQueueExtension purgeQueue = PurgeQueueExtension
+            .purgeIfTestFailed(sleeper, INGEST_JOB_QUEUE_URL, COMPACTION_JOB_QUEUE_URL);
```

Comment:
Does this test need to purge the ingest job queue on failure if it's performing a direct ingest?